content/using-atom/sections/moving-in-atom.md

```diff
@@ -40,6 +40,111 @@ You can also move directly to a specific line (and column) number with <kbd clas
 
 ![Go directly to a line](../../images/goto.png "Go directly to a line")
 
+
+#### Additional Movement and Selection Commands
+
+Atom also has a few movement and selection commands that don't have a keyboard shortcut mapped to it.   You can access these commands from the [Command Palatte](http://flight-manual.atom.io/getting-started/sections/atom-basics/#command-palette), but if you find yourself using a command that doesn't have a keyboard shortcut often, have no fear!  You can easily add a entry to your `keymaps.cson` to create a keyboard shortcut.  
```

Comment:
> that don't have a keyboard shortcut mapped to it

that don't have key bindings by default.

Reply:
Looks like we use one word `keybindings` on this page ✏️ so:

> that don't have keybindings by default.

Also /s/"easily add a entry"/"easily add an entry".

Reply:
/s/keymaps.cson/keymap.cson I think?

Also maybe link `keymap.cson` to http://flight-manual.atom.io/using-atom/sections/basic-customization/#customizing-keybindings?  Probably ok without linking though.

Reply:
I'm trying to recommend these terms for use around keymaps: https://github.com/atom/flight-manual.atom.io/blob/master/CONTRIBUTING.md#terminology

Reply:
> Command Palatte

Command Palette